modules/prediction/evaluator/evaluator_manager.cc

```diff
@@ -54,6 +58,23 @@ bool IsTrainable(const Feature& feature) {
   return true;
 }
 
+void GroupObstaclesByPriority(const int obstacle_id,
+    ObstaclesContainer* const obstacles_container,
+    PriorityObstacleMap* const priority_obstacle_map) {
+  Obstacle* obstacle_ptr = obstacles_container->GetObstacle(obstacle_id);
+  if (obstacle_ptr == nullptr) {
+    AERROR << "Null obstacle [" << obstacle_id << "] found";
+    return;
+  }
+  const Feature& feature = obstacle_ptr->latest_feature();
+  const ObstaclePriority& priority = feature.priority();
+  if (priority.priority() == ObstaclePriority::IGNORE) {
+    ADEBUG << "Skip ignored obstacle [" << obstacle_id << "]";
+    return;
+  }
+  (*priority_obstacle_map)[priority.priority()].push_back(obstacle_ptr);
```

Comment:
Could you try `priority_obstacle_map->emplace(priority.priority(), obstacle_ptr)`